lib/privs.c

```diff
@@ -496,26 +501,26 @@ struct zebra_privs_t *_zprivs_raise(struct zebra_privs_t *privs,
 	int save_errno = errno;
 	struct zebra_privs_refs_t *refs;
 
-	if (!privs)
-		return NULL;
-
 	/*
 	 * Serialize 'raise' operations; particularly important for
 	 * OSes where privs are process-wide.
 	 */
-	frr_with_mutex (&(privs->mutex)) {
-		/* Locate ref-counting object to use */
-		refs = get_privs_refs(privs);
+	pthread_mutex_lock(&zprivs_mutex);
+	if (!privs)
+		return NULL;
+	;
```

Comment:
The stray `;` in aisle 9 is looking for its parents, can they please come to the customer service counter? :stuck_out_tongue_closed_eyes:

Reply:
fixed